regression/cbmc-java/static_init_order/static_init_order.java

```diff
@@ -0,0 +1,37 @@
+public class static_init_order
+{
+  public static void main()
+  {
+    C object2 = new C();
+    B object  = new B();
+    // as per the Java Virtual Machine Specification,
+    // section 5.5, p. 35 we expect the static initializer of
+    // the parent class to be called before the static initializer
+    // of the class in question.
+    if(object2.x == 20)
+        // order of Init:  C.clint, A.clint, B.clint
+        assert(true);
```

Comment:
I don't think asserting true is useful.
Actually if we have 2 `assert false` and jbmc says the first can fail but not the second, I find this more interesting.

Reply:
Agreed, will address this.